.github/workflows/CI.yaml

```diff
@@ -0,0 +1,74 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - "*"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Install Python Dependencies
+      run: |
+        pip install -r source/requirements.txt
+    - name: Install LaTeX
+      run: |
+        sudo apt install -y texlive-latex-recommended texlive-fonts-recommended texlive-latex-extra latexmk texlive-lang-greek texlive-luatex texlive-xetex texlive-fonts-extra dvipng librsvg2-bin
+    - name: Build HTML
+      run: |
+        make html
+    - name: Build PDF
+      run: |
+        make latexpdf
+    - name: Archive HTML
+      uses: actions/upload-artifact@v2
+      with:
+        name: frc-docs-html
+        path: build/html/
```

Comment:
This was before the make pdf call on azure to make sure the html artifact got uploaded even if the pdf failed.